osscameroon/yotas
# Dev models

#### This pull request add models generated by using the migrations description

**Why ?**
We need to interact with models to bind data stored in bd with logic

**How ?**
By adding struct for each table present in [backend/migrations/001_initial.sql](https://github.com/osscameroon/yotas/pull/38/files#diff-56b8a16c65214d799a928a0f5801581f4982dd6781a6dd7058d8619f4cb05d2c)


**Steps to verify:**
Verify if each table present in [backend/migrations/001_initial.sql](https://github.com/osscameroon/yotas/pull/38/files#diff-56b8a16c65214d799a928a0f5801581f4982dd6781a6dd7058d8619f4cb05d2c) are created.

** Screenshots (optional)
